eth_keys.nim

```diff
@@ -139,3 +139,15 @@ proc toCanonicalAddress*(pubkey: PublicKey): array[20, byte] =
   ## Convert public key to canonical address.
   var hash = keccak256.digest(pubkey.getRaw())
   copyMem(addr result[0], addr hash.data[12], 20)
+
+proc `$`*(pubkey: PublicKey): string =
+  ## Convert public key to hexadecimal string representation.
+  result = "0x" & toHex(pubkey.getRaw(), true)
```

Comment:
I would prefer not to have `0x` in front. It's easy to add outside if needed, but it's harder (less efficient) to trim it away if it's not.